docs/content/manual/manual.yml

```diff
@@ -1772,6 +1772,27 @@ sections:
             input: '["fo", "foo", "barfoo", "foobar", "foob"]'
             output: ['["fo","","bar","foobar","foob"]']
 
+      - title: "`trim`"
+        body: |
+
+          Trim leading and trailing whitespace in input.
+
+          Whitespace is
+          `" "`,
+          `"\n"`
+          `"\t"`,
+          `"\r"`,
+          `"\f"`,
+          `"\u000b"` (vertical tab),
+          `"\u0085"` (next line) and
+          `"\u00a0"` (no-break space). These are the whitespace characters in
```

Comment:
Is this exhaustive for Unicode, or just Latin scripts in Unicode?  If the latter, why not be more exhaustive?

Reply:
Is only whitespace from basic block and latin-1, so not exhaustive. Reason is mostly to match what other implementations do. There are quite a lot of other whitespace characters in other blocks, wikipedia has a good list https://en.wikipedia.org/wiki/Whitespace_character.

Reading about PCRE's `\s`-class it seem to match something similar.

Reply:
Rust `trim`/`is_whitespace` uses characters with `White_Space` property https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt maybe more resonable?

Reply:
Maybe we should document that the list is not stable, that we may add to it.

Reply:
> Rust `trim`/`is_whitespace` uses characters with `White_Space` property https://www.unicode.org/Public/UCD/latest/ucd/PropList.txt maybe more resonable?

I do like that better, yeah, especially in light of @itchyny's comment.

Reply:
Wait for utf8proc to be included for upcase/downcase?

Reply:
Update to `White_Space` but i guess we can wait also